src/python/dxpy/dx_extract_utils/cohort_filter_payload.py

```diff
@@ -0,0 +1,141 @@
+import copy
+
+
+def generate_pheno_filter(values, entity, field, filters):
+    filters = copy.deepcopy(filters)
```

Comment:
Just heads up that deepcopy is very slow for more complicated filters. Maybe better idea is to create a new filters hash and copy over things that you need will have in filters.

Reply:
We don't need to copy this dict. Will just delete this line.